cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -582,17 +605,47 @@ protected ServicePort createServicePort(String name, int port, int targetPort, I
         return servicePort;
     }
 
+    protected PersistentVolumeClaim createPersistentVolumeClaimTemplate(String name, PersistentClaimStorage storage) {
+        Map<String, Quantity> requests = new HashMap<>();
+        requests.put("storage", new Quantity(storage.getSize(), null));
```

Comment:
Might there be other requests which users might want to configure on the PVCs?

Reply:
There are apparently. It seems that for example IBM storage supports IOPS configuration next to the size (whereas in AWS this is for example done through storage class and not PVC). 

But:
* The actual fields are not specified anywhere (It is using the same resources object as are used for CPU and memory). So we do not know what to support.
* Our current CRDs are not using the resources object which would give us the flexibility but instead have just a single specific field for size. This is where this code comes from.

This is not really something what I changed somehow in this PR and I would leave that for another PR. We should probably think whether we should move to using the Resources in our CRDs here as well as part of v1beta1. Another think is that also ephemeral storage might now have a size limit - that is something we might want to have a look at as well in the future.

Reply:
FYI: I opened issue #1504 for this.